react/Popover.tsx

```diff
@@ -13,10 +13,17 @@ import handleContainerStyle, { RestoreFn } from './modules/handleContainerStyle'
 
 type TransitionComponentType = 'fade'
 
+interface Offsets {
+  distance: number
+  skidding: number
+}
+
 interface Props {
   role?: string
-  placement: Placement
+  arrow?: boolean
```

Comment:
Any chance of this becoming an `Enum`? Just bringing this up, I can't think of a use case.
I would at least make this responsive, what do you think?